libnymea-zigbee/backends/ti/zigbeenetworkti.cpp

```diff
@@ -291,7 +293,7 @@ void ZigbeeNetworkTi::processGreenPowerFrame(const Zigbee::ApsdeDataIndication &
 
 QByteArray ZigbeeNetworkTi::encryptSecurityKey(quint32 sourceId, const QByteArray &securityKey)
 {
-#if (QCA_VERSION >= QCA_VERSION_CHECK(2, 2, 0))
```

Comment:
This will probably fail to build on older distributions as you're dropping the check for version 2.2.
Ideally packagesExist would check that, but I am not aware of packagesExist offering that feature. So I guess we'll just do `#if !defined (DISABLE_QCA ) && (QCA_VERSION >= QCA_VERSION_CHECK(2,2,0))` here